doc/merging.rst

```diff
@@ -0,0 +1,56 @@
+How to merge a branch after approving the PR
+============================================
+
+Requirements
+--------------
+
+- A remote called *upstream* pointing at the central IC repository.
+
+- A remote configured for the author of the PR.
+
+- After the PR is approved, make sure the merge should be done (asking the authors if they didn't get in touch with you).
+
+
+Steps
+-------
+
+#. Fetch *upstream/master*.
+
+#. Reset or rebase your master branch to *upstream/master*.
```

Comment:
Safer to leave `reset` only in case the merger was erroneously developing in master?

Reply:
If the merger was developing on `master` and resets it ... the work on it is lost! (Modulo commits that are in reflogs, yadda, yadda.)

Perhaps we should instruct the merger to create a `merge-pr` branch specifically for this purpose, then we don't have to make any assumptions about the local master.

Reply:
But this `merge-pr` branch would substitute the `master` branch or the branch created on top of the PR branch to merge?

Reply:
The process requires two branches, one tracking `upstream/master` and one tracking `author/feature`. It seems fairly safe for the merger to use a local `feature` to track `author/feature`. We're addressing the possibility raised by @mmkekic that the merger might have been doing something silly with local `master` so it would replace local `master`.

Perhaps it's not worth it. Perhaps we should just stick to `master`. I don't know, these are instructions for *people* and people are so complicated and do so many strange things! We can't possibly anticipate them all.

Reply:
I don't think it 's worth it. People should know that if they want to contribute, they should make a master independent branch :)

Reply:
Maybe simply saying 'Make sure your local `master` and `upstream/master` are pointing to the same commit' or some more precise/descriptive version of that?

Reply:
Yes, I like it!